crates/circuit/src/dag_circuit.rs

```diff
@@ -6169,11 +6171,11 @@ impl DAGCircuit {
     /// * node_index: The node in the DAGCircuit to replace.
     /// * other: The replacement DAGCircuit.
     /// * qubit_map: A mapping from the replacement DAGCircuit qubits to the replaced node's qargs.
-    ///         If None, trivial mapping will be used.
+    ///   If None, trivial mapping will be used.
```

Comment:
Hey, this isn't to do with PyO3 ;)

Reply:
I had to fix all the clippy warnings as part of the changes. This was in there for me locally...